packages/components/src/components/swipeable-notification/swipeable-notification.tsx

```diff
@@ -47,11 +48,7 @@ const SwipeableNotification = ({
     );
     const getDisplayedTime = () => {
         if (!seconds && seconds !== 0) return;
-        return seconds < 10 ? (
-            <Localize i18n_default_text='now' />
-        ) : (
-            <Localize i18n_default_text='{{seconds}}s ago' values={{ seconds }} />
-        );
+        return seconds < 10 ? <Localize i18n_default_text='now' /> : getTimeSince(Number(timestamp) * 1000);
```

Comment:
For our use case in DTrader, it will always be some 'XXs ago' but for better reusability of SwipeableNotification, I've added `getTimeSince`, so that it can work with various time units.